doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -74,18 +74,27 @@ Identifiers
 
 Numerals
   Numerals are sequences of digits with an optional fractional part
-  and exponent, optionally preceded by a minus sign. :n:`@int` is an integer;
-  a numeral without fractional or exponent parts. :n:`@num` is a non-negative
+  and exponent, optionally preceded by a minus sign. Hexadecimal numerals
+  start with ''0x' or ''0X'. :n:`@int` is an integer;
```

Comment:
You probably meant to use backticks instead:

```suggestion
  start with ``0x`` or ``0X``. :n:`@int` is an integer;
```

Reply:
Indeed, done, thanks!